persistence/persistence-api/src/main/java/org/cbioportal/persistence/DiscreteCopyNumberRepository.java

```diff
@@ -44,7 +44,7 @@ BaseMeta fetchMetaDiscreteCopyNumbersInMolecularProfile(String molecularProfileI
                                                                           List<Integer> entrezGeneIds, 
                                                                           List<Integer> alterations);
 
-    List<CopyNumberCountByGene> getPatientCountByGeneAndAlterationAndPatientIds(String molecularProfileId,
+    List<CopyNumberCountByGene> getPatientCountInMultipleMolecularProfiles(List<String> molecularProfileIds,
```

Comment:
@kalletlak im not sure if this function name really indicates what it does. It takes as arguments genetic alteration + molecularprofileids. So maybe calling it `getPatientCountByMolecularProfilesGeneAndAlteration` is more appropriate? Or just `getPatientCount`. The input arguments are indicative enough of expected input?

Reply:
true, but I was following the same conventions what's there for `getSampleCountInMultipleMolecularProfiles`

Reply:
Ok makes sense